### ISSUE TITLE
System tray not visible on some Linux distributions

### ISSUE THREAD

**Issue by ManfredKarrer** [bisq-network/bisq#350]
"First start was ok (testnet synced, Tor connection established) but 2rd and follow attempts failed with error messages stating that running few Bitsquare at the same time is impossible. I guess when I click on Close button of running Bitsquare, it doesn't close, but minimize to systray instead. But the problem is that seems like Java using xembed systray icons, and Plasma 5.4 (DE of Kubuntu 15.10) is phasing out dependency on X11 technologies/protocols, including xembed (because DE developers implementing support for new protocol Wayland for communication between applications and display server, they keep only functional that works with this new protocol, and xembed happen to work only with old one - X11; also there is number of issues with xembed). So xembed systray icons doesn't work for me and when I click close on Bitsquare window I thought it's really closed while in fact it continue running (still not sure if it's really the case because I doesn't know how to check this). 
If I was right, then issue in Java, that using xembed directly, instead of relying on toolkit (in case of Java on Linux it's GTK3, for example save dialog in Bitsquare is GTK3-style) for interaction with DE systray. There is more details.
Also libappindicator support could be implemented in Java directly, but not sure if this necessary, if they could use implemented GTK3 libappindicator support.
Also as far I know other DE is also phase out xembed support, for example Gnome Shell."

-> We deactivate that "keep running in background" behaviour for the moment until those issues are solved. Need a information alert also to tell the user about that behviour.

**Comment by bitspill** [comment]
Steam has had the same issue since at least January 2013 and their approach was to substitute the close action with minimize and utilize an environment variable to enable closing to the sys tray for users who know it will work on their system.

https://github.com/ValveSoftware/steam-for-linux/issues/1025

**Comment by ManfredKarrer** [comment]
Thanks for the hint!

**Comment by qtlin** [comment]
@ManfredKarrer System tray still not supported? Should I open a new issue?
```
[AWT-EventQueue-0] ERROR i.b.g.a.BisqApp: throwableMessage= The system tray is not supported on the current platform. 
java.lang.UnsupportedOperationException: The system tray is not supported on the current platform.
```
Ubuntu 16.04

**Comment by erciccione** [comment]
This problem is still current. The system tray is not visible since years for me, would be nice to have this fixed because it's bad UX. Additionally, the tray icon appear as an opened page on Debian, forcing me to right click the main icon, choose the actual bisq interface, and finally click to visualize it. I also get a "the system tray is not supported on this system" warning every time my laptop turns of the screen to save energy. It's very annoying.

![bisq](https://user-images.githubusercontent.com/28106476/63022057-8b699a80-be91-11e9-9556-ec7bb60c2ab0.png)


The problem shows in both Debian Sid and Buster (Gnome) and was present in Stretch.

**Comment by premek** [comment]
the error popup removed in #4874

**Comment by ripcurlx** [comment]
Closing as complete.

**Comment by premek** [comment]
my comment refererd only to removing the error message "The system tray is not supported on the current platform"

**Comment by premek** [comment]
It seems to me that the icon is misplaced on Xorg but it's at the right place on Wayland

Wayland (correct):
![Screenshot from 2020-12-09 20-14-15](https://user-images.githubusercontent.com/1145361/101692613-c1606400-3a70-11eb-9b9e-32b3b926201e.png)

Xorg (icon in top left corner but clickable at the correct position in the top bar at the right side)
![Screenshot from 2020-12-09 22-44-29](https://user-images.githubusercontent.com/1145361/101692611-c02f3700-3a70-11eb-962c-781539a6bc8e.png)

**Comment by ripcurlx** [comment]
This might improve when switching to the latest Java runtime and packager in #4242.

**Comment by cd2357** [comment]
Should be fixed in the most recent release (v1.6.4).
